Test/ContentPipeline/IntermediateDeserializerTest.cs

```diff
@@ -266,27 +266,6 @@ public void MathTypes()
         }
 
         [Test]
-        public void PrimitiveTypes()
-        {
-            DeserializeCompileAndLoad<PrimitiveTypes>("18_PrimitiveTypes.xml", primitiveTypes =>
-            {
-                Assert.AreEqual('A', primitiveTypes.Char);
-                Assert.AreEqual(127, primitiveTypes.Byte);
-                Assert.AreEqual(-127, primitiveTypes.SByte);
-                Assert.AreEqual(-1000, primitiveTypes.Short);
-                Assert.AreEqual(1000, primitiveTypes.UShort);
-                Assert.AreEqual(-100000, primitiveTypes.Int);
-                Assert.AreEqual(100000, primitiveTypes.UInt);
-                Assert.AreEqual(-10000000, primitiveTypes.Long);
-                Assert.AreEqual(10000000, primitiveTypes.ULong);
-                Assert.AreEqual(1234567.0f, primitiveTypes.Float);
-                Assert.AreEqual(1234567890.0, primitiveTypes.Double);
-                Assert.AreEqual(null, primitiveTypes.NullChar);
-                Assert.AreEqual(' ', primitiveTypes.NotNullChar);
-            });
-        }
-
-        [Test]
```

Comment:
Try to avoid moving functions around within an existing source file.  It makes it very difficult to track what the actual changes are on the GitHub diff view as it will show as the entire function as been deleted and re-added elsewhere.